e2e-tests/development-runtime/cypress/integration/eslint-rules/limited-exports-page-templates.js

```diff
@@ -0,0 +1,14 @@
+if (Cypress.env("HOT_LOADER") === `fast-refresh`) {
```

Comment:
I presume we do run all the tests with and without all these env flags set as well?

Reply:
Since the change is only really relevant for fast-refresh I opted to only test this for that. In Michal's PR we'll be conditionally loading the rules when the ENV var is set